azure-nextgen-py-static-website/Pulumi.yaml

```diff
@@ -0,0 +1,11 @@
+name: azure-nextgen-py-static-website
+runtime:
+  name: python
+  options:
+    virtualenv: venv
+description: An example of a static website hosted on Azure Blob Storage + Azure CDN.
+template:
```

Comment:
I guess you copied this from the other example. But do you know if this actually makes sense here? I thought `template` would only apply to `pulumi new`?

Reply:
Will remove.